app/src/main/AndroidManifest.xml

```diff
@@ -115,7 +115,7 @@
         <activity android:name=".usescases.reservedValue.ReservedValueActivity" />
         <activity
             android:name=".usescases.eventsWithoutRegistration.eventCapture.EventCaptureActivity"
-            android:configChanges="keyboardHidden|orientation|screenSize"
```

Comment:
I removed the `orientation` parameter from the list because I prefer letting Android handle orientation changes automatically. This prevents issues like the one we experienced in EventCaptureActivity when the app didn't respond to portrait-to-landscape changes. By removing the parameter, we allow Android to handle orientation changes, resolving the problem we encountered